ckan/templates/group/about.html

```diff
@@ -3,14 +3,22 @@
 {% block subtitle %}{{ _('About') }} - {{ super() }}{% endblock %}
 
 {% block primary_content_inner %}
-  <h1>{% block page_heading %}{{ c.group_dict.display_name }}{% endblock %}</h1>
+  <h1>
+    {% block page_heading %}
+      {% if c.group_dict.title %}
```

Comment:
this doesn't look right. The translated version will be stored in `title_translated` so checking the title relies on the correct validator being set up for `title`. Why don't you follow the `h.get_translated(c.group_dict, 'title') or c.group_dict.display_name` pattern from below instead?